ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ContractStateEventsReader.scala

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.appendonlydao.events
+
+import java.io.InputStream
+import java.sql.Connection
+import java.time.Instant
+
+import anorm.SqlParser.{binaryStream, int, long}
+import anorm._
+import com.daml.ledger.participant.state.v1.Offset
+import com.daml.lf.data.Ref
+import com.daml.lf.transaction.GlobalKey
```

Comment:
`com.daml.lf.transaction.GlobalKey` is mapped within this project to `Key`, no need to import it

Reply:
Good catch! Thanks 👍